app/models/project.rb

```diff
@@ -64,6 +64,7 @@ class Project < ApplicationRecord
                                 reject_if: :all_blank
 
   scope :visible_to, (lambda do |user, team|
+                        # Team owners see all projects in the team
```

Comment:
In this scope, should we call `team.projects`? If we call just `Project.visible_to(user, team)`, this will return all projects, regardless of team, if the user is an owner of the team we're passing to the scope.

Reply:
Nope, that will brake chaining of other conditions